docs/cow-protocol/reference/contracts/programmatic/cow-amm.md

```diff
@@ -4,6 +4,16 @@ sidebar_position: 1
 
 # CoW AMM
 
+:::warning
```

Comment:
```suggestion
:::caution
```

I would generally classify warning admonitions as required when there is a high likelihood of loss of funds. I don't think this the case here so we can downgrade this to caution :smiley: